### ISSUE TITLE
Really hoping to see support for this on iOS 9 

### ISSUE THREAD

**Issue by eweiman** [b3ll/Ignition#43]
I know this is a 'when you have time' kind of project, and I'm really sad I updated my iPad Mini Retina from 8.1 to 8.4 before checking to see if this worked on 8.4 ... As Apple keeps changing things around with CarPlay its expected that there are issues that need to be worked out.

With Apple supporting more CarPlay display types I was hoping that the change may make this easier to support with iOS 9. 

Would it be easier to have a second device be required rather than having the device run CarPlay with itself?
Basically make it a CarPlay receiver that communicated back to a phone, which I would think would resolve (or help resolve) the missing asset issues you had mentioned on twitter a while back.
